src/documentation/about/tao.malloynb

```diff
@@ -11,16 +11,16 @@ Our primary users will all be familiar with SQL. We should make their life no ha
 For example `name is expression`, vs `expression as name`. In SQL the naming on an expression sits as a casual afterthought "oh by the way, give this really important expression a name". In Malloy, the name of a thing is important, you are building complex things from smaller pieces. When you look at a model, you will often want to scan the file looking for names, they belong on the left hand side.
 
 
-However, we in general try to have the "feel" of SQL. We use `()` for structuring instead of `{}` like LookML or JavaScript do, or indentation like Python does. We use `is` and `define` instead of `:` like LookML or `=` like JavaScript. We use SQL words for things (like `join`) where it makes sense.
+However try to keep to the "feel" of SQL while incorporating useful syntax from other languages. We use `()` for structuring instead of `{}` (LookML/JavaScript), or whitespace/indentation (Python). When a concept or function is particular to query languages we try to fall back to use SQL words (i.e. `join`, `aggregate`) but do not introduce whitespace syntax (i.e. malloy's `group_by` vs SQL's `GROUP BY`)
```

Comment:
Most of this is trying to improve the general "flow" of the sentence.

The last bit regarding "falling back to SQL words" is inferred from what I've seen of the language and should be confirmed by an expert.